foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/logging/LogCategory.java

```diff
@@ -61,15 +62,16 @@
     METAMODEL(  (byte)0x0A, SessionLog.METAMODEL),
     MISC(       (byte)0x0B, SessionLog.MISC),
     MONITORING( (byte)0x0C, SessionLog.MONITORING),
-    PROCESSOR(  (byte)0x0D, SessionLog.PROCESSOR),
-    PROPAGATION((byte)0x0E, SessionLog.PROPAGATION),
-    PROPERTIES( (byte)0x0F, SessionLog.PROPERTIES),
-    QUERY(      (byte)0x10, SessionLog.QUERY),
-    SEQUENCING( (byte)0x11, SessionLog.SEQUENCING),
-    SERVER(     (byte)0x12, SessionLog.SERVER),
-    SQL(        (byte)0x13, SessionLog.SQL),
-    TRANSACTION((byte)0x14, SessionLog.TRANSACTION),
-    WEAVER(     (byte)0x15, SessionLog.WEAVER);
+    MOXY(       (byte)0x0D, SessionLog.MOXY),
```

Comment:
moxy as a _new_ category should be using some _new_ id (0x16) and not redefine existing one (0x0D) and shifting others. This is in a public package and can eventually break existing apps...

Reply:
Hello @lukasj . I've Added MOXY category in this way because I was guided by the following commit https://github.com/eclipse-ee4j/eclipselink/commit/e73f1d759dff45e032733599fdc89adf0848d6b2#diff-1bd332bafe844727dd0be92daa2fd1d2L59 . Is it incorrect as well?

Reply:
oh well, let's keep it your way then...